torch/fx/graph.py

```diff
@@ -291,16 +291,15 @@ def __exit__(self, type, value, tb):
 
 class _node_list:
     def __init__(self, graph: "Graph", direction: str = "_next"):
-        assert direction in ["_next", "_prev"]
+        assert direction in ("_next", "_prev")
         self.graph = graph
         self.direction = direction
 
     def __len__(self):
         return self.graph._len
 
     def __iter__(self):
-        assert self.direction == "_prev" or self.direction == "_next"
```

Comment:
You could make direction a property and have this verification on the property setter.

Reply:
The runtime cost of doing this would offset the perf gain from this PR (properties are much slower).

Reply:
Fair, just go with the literal approach then, it's will only affect the typing system.